csharp/extractor/Semmle.Extraction.CSharp/Entities/Modifier.cs

```diff
@@ -111,6 +111,9 @@ public static void ExtractModifiers(Context cx, TextWriter trapFile, IEntity key
                 if (nt is null)
                     throw new InternalError(symbol, "Symbol kind is inconsistent with its type");
 
+                if (nt.IsRecord)
```

Comment:
This requires, in principle, a DB upgrade script, since DBs created before this extractor change should have `record` modifiers populated whenever a clone method exists. Upgrades without DB scheme changes can be done by making a dummy change to the DB scheme, namely updating the date in the header of the file.

Reply:
Aha, I see. So providing a "dummy" upgrade script (and now also downgrade script) will re-create the database to ensure the consistency with the change of semantics.

Reply:
Nice catch btw!

Reply:
I just realized that this can't be done, since we can't create a new `@modifier` entity for `record` in an upgrade script. So let's just not touch the DB scheme on this PR.